src/Command/Source/SourceUpdateImageMetaCommand.php

```diff
@@ -53,12 +54,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $queryStr = 'SELECT s FROM App\Entity\Source s WHERE s.matchId='.$identifier;
         }
         $query = $this->em->createQuery($queryStr);
-        $iterableResult = $query->iterate();
-        foreach ($iterableResult as $row) {
-            $source = $row[0];
 
+        /** @var Source $source */
+        foreach ($query->toIterable() as $source) {
             $item = new VendorImageItem();
-            $item->setOriginalFile($source->getOriginalFile());
+
+            $originalFile = $source->getOriginalFile();
+            if (null !== $originalFile) {
```

Comment:
Not sure that is can happen...

Reply:
Maybe not. But Psalm should shut the f*ck up!